public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -121,16 +124,19 @@ const ui = {
   newRuleButton: byRole('link', { name: 'New alert rule' }),
 
   editGroupModal: {
-    namespaceInput: byLabelText('Namespace'),
-    ruleGroupInput: byLabelText('Rule group'),
-    intervalInput: byLabelText('Rule group evaluation interval'),
+    namespaceInput: byRole('textbox', { hidden: true, name: /namespace/i }),
+    ruleGroupInput: byRole('textbox', { name: 'Evaluation group', exact: true }),
+    intervalInput: byRole('textbox', {
+      name: /Rule group evaluation interval Evaluation interval should be smaller or equal than 'For' values for existing rules in this group./i,
+    }),
```

Comment:
this one looks different to the one below; which one did you wnat to keep?

also we need to change the preposition here--it's always equal to and not than :)

Reply:
@brendamuir, don't pay attention to files *.test.tsx, as these are tests files that depend on the components texts. We'd better check the `*.tsx `files.